test/sql/updates/post.catalog.sql

```diff
@@ -45,20 +45,18 @@ SELECT count(*)
 -- The list of tables configured to be dumped.
 SELECT unnest(extconfig)::regclass::text, unnest(extcondition) FROM pg_extension WHERE extname = 'timescaledb' ORDER BY 1;
 
--- Show dropped chunks
-SELECT id, hypertable_id, schema_name, table_name, dropped
-FROM  _timescaledb_catalog.chunk c
-WHERE c.dropped
-ORDER BY c.id, c.hypertable_id;
-
 -- Show chunks that are not dropped and include owner in the output
 SELECT c.id, c.hypertable_id, c.schema_name, c.table_name, c.dropped, cl.relowner::regrole
 FROM  _timescaledb_catalog.chunk c
 INNER JOIN pg_class cl ON (cl.oid=format('%I.%I', schema_name, table_name)::regclass)
-WHERE NOT c.dropped
+WHERE c.dropped IS FALSE
```

Comment:
What's the difference between old and new, `c.dropped` can be null?

Reply:
It cannot... just a cosmetic thing... sometimes I have some OCD regarding some coding styles... hehehehe... will preserve the original because it will not improve anything here and just add noise to repo history.